### PR TITLE
Update dependencies for Bazel 0.8.0

### PR DESCRIPTION
**Pull request by ejona86**
The updated versions themselves now build with newer Bazels. But Bazel
also no longer needs as many repositories defined, as C++, protoc, and
Java repositories are assumed to be shared.

### CODE REVIEW

File: WORKSPACE [hunk 1]
```diff
@@ -1,36 +1,18 @@
...
 )
 
 # java_lite_proto_library rules implicitly depend on @com_google_protobuf_javalite//:javalite_toolchain,
 # which is the JavaLite proto runtime (base classes and common utilities).
 http_archive(
     name = "com_google_protobuf_javalite",
-    urls = ["https://github.com/google/protobuf/archive/45ff30697af6590a927c73584351f2670768f719.zip"],
-    strip_prefix = "protobuf-45ff30697af6590a927c73584351f2670768f719",
-    sha256 = "2244bc5077087be2e5ca8fab9cb4b614443b572cedd83a01d773d3c6c971635c",
+    sha256 = "d8a2fed3708781196f92e1e7e7e713cf66804bd2944894401057214aff4f468e",
```
Comment (cgrushko) [inline WORKSPACE:15]: did Buildifier rearrange the lines?

**Review by cgrushko — COMMENTED** [review]
Awesome! 
Just in case - did you successfully build the repo with this change?

File: WORKSPACE (hunk 1 above)
Comment (ejona86) [reply WORKSPACE:15]: Yeah.

**Comment by ejona86** [comment]
Yes, I successfully built the repo (`bazel build ...`) with this change using Bazel 0.8.0.

**Pull request merged**